openrtb/openrtbutil/client_functional_test.go

```diff
@@ -194,13 +194,14 @@ func TestClientDoShouldRespondNoBid(t *testing.T) {
 			}
 		}
 
-		ctx, _ := context.WithTimeout(context.Background(), tmax)
+		ctx, cancel := context.WithTimeout(context.Background(), tmax)
```

Comment:
defer cancel()

Reply:
I'd like to. But original code use t.Fatal to force exit in one path, so defer can be passed in that case.

Reply:
In this case, the defer function would be called before panic()

Reply:
Yes, you are right. testing Fatal has different behavior against log Fatal and I had thought they are same.